apps/designer/app/designer/features/style-panel/shared/parse-css-value.ts

```diff
@@ -2,9 +2,11 @@ import hyphenate from "hyphenate-style-name";
 import { units } from "@webstudio-is/react-sdk";
 import type { StyleProperty, StyleValue, Unit } from "@webstudio-is/react-sdk";
 
-const unitRegex = new RegExp(`${units.join("|")}`);
+const unitRegex = new RegExp(
+  `${(units as unknown as string[]).concat("number").join("|")}`
```

Comment:
same here with assertions

Reply:
`.concat` is a generic function and its type definitions only allow you to pass items of the type contained in the source array. Basically, for a given `T[]` concat can only produce `T[]`. Your `T` here is a list of some string literals (representing units) and `"number"` is not one of them thus it's not accepted as a valid argument because it's not assignable to `T`.

As a workaround for this I recommend using spread - which allows you to create new array types out of the input "arguments":
```ts
const unitRegex = new RegExp(`${[...units, "number"].join("|")}`);
```